commit 48eb4034563e29dd3fba7310106d929811da32eb
Author: Anatol Pomozov <anatol.pomozov@gmail.com>
Date:   Tue Jun 11 21:30:13 2019 -0700

    Revert "Update System V and System D installers to define user, group and runtime directory as necessary"
    
    This reverts commit 15c376ad57629dad16f1120dbf4c080f7de48475.

diff --git a/Makefile.am b/Makefile.am
index 2bbc387..99db873 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -185,14 +185,10 @@ endif
 endif
 endif
 if INSTALL_SYSTEMV
-	getent group shairport-sync &>/dev/null || groupadd -r shairport-sync >/dev/null
-	getent passwd shairport-sync &> /dev/null || useradd -r -M -g shairport-sync -s /usr/bin/nologin -G audio shairport-sync >/dev/null
 	[ -e $(DESTDIR)$(sysconfdir)/init.d ] || mkdir -p $(DESTDIR)$(sysconfdir)/init.d
 	[ -f $(DESTDIR)$(sysconfdir)/init.d/shairport-sync ] || cp scripts/shairport-sync $(DESTDIR)$(sysconfdir)/init.d/
 endif
 if INSTALL_SYSTEMD
-	getent group shairport-sync &>/dev/null || groupadd -r shairport-sync >/dev/null
-	getent passwd shairport-sync &> /dev/null || useradd -r -M -g shairport-sync -s /usr/bin/nologin -G audio shairport-sync >/dev/null
 	[ -e $(DESTDIR)$(systemdsystemunitdir) ] || mkdir -p $(DESTDIR)$(systemdsystemunitdir)
 	[ -f $(DESTDIR)$(systemdsystemunitdir)/shairport-sync.service ] || cp scripts/shairport-sync.service $(DESTDIR)$(systemdsystemunitdir)
 endif
